recipes/spirv-headers/all/conanfile.py

```diff
@@ -30,6 +31,8 @@ def source(self):
         extracted_dir = "SPIRV-Headers-" + self.version
         if self.version == "1.5.1":
             extracted_dir = extracted_dir + ".corrected"
+        if self.version.startswith('cci.'):
+            extracted_dir = glob.glob(self.name + "-*/")[0]
         os.rename(extracted_dir, self._source_subfolder)
```

Comment:
`glob` can be avoided, and `source()`simplified with:

```python
def source(self):
    tools.get(**self.conan_data["sources"][self.version],
              destination=self._source_subfolder, strip_root=True)
```

Reply:
Done, thanks :)